components/esp_lvgl_port/src/lvgl9/simd/lv_color_blend_to_argb8888_aes3.S

```diff
@@ -0,0 +1,298 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "sdkconfig.h"
+
+#if CONFIG_IDF_TARGET_ESP32S3
+
+// This is LVGL ARGB8888 simple fill for ESP32S3 processor.
+    .text
+    .align  4
+    .global lv_color_blend_to_argb8888_esp32_aes3
+    .global .lv_color_blend_to_argb8888_esp32_ae32_body
+    .type   lv_color_blend_to_argb8888_esp32_aes3,@function
```

Comment:
What is the difference between `aes3` and `ae32` functions? What do the acronyms mean?

Reply:
I used acronyms used in [esp-dsp](https://github.com/espressif/esp-dsp/blob/master/modules/matrix/mul/include/dspm_mult.h#L44)
`aes3` is assembly implementation for esp32s3
`ae32` is assembly implementation for esp32
`ansi` is a gneral C code

Do you think it would make more sense if we changed it somehow?

Reply:
I think the target names (`esp32`, `esp32s3`) could be more understandable to other developers looking at this code. We don't have to deal with the ANSI C implementation here, as it is already included into LVGL, so what remains is just to distinguish between the chips.

Another option could be to use some terms to describe the instruction sets, for example `xtensa_base`, `xtensa_pie`, `riscv_pie`, etc. But we might run into the limitations of this approach if we have more RISC-V chips with slightly different sets of PIE (processor instruction extensions).

Reply:
`*_esp32` and `*_esp32s3` would make more sense then. 

Also, `xtensa_base` would not really make sense here, since the esp32s2 does not have zero overhead loops ( `loopnez` opcode in assembly) if I am not mistaken. And  the `loopnez` is used in each assembly function here.